packages/x-date-pickers/src/DateCalendar/tests/validation.CalendarPicker.test.tsx

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
 import { expect } from 'chai';
 import { screen, fireEvent } from '@mui/monorepo/test/utils';
-import { CalendarPicker, CalendarPickerProps } from '@mui/x-date-pickers/CalendarPicker';
+import { DateCalendar, DateCalendarProps } from '@mui/x-date-pickers/DateCalendar';
```

Comment:
Should probably rename the file to `validation.DateCalendar.test.tsx` instead as well? 🤔